capella_model_explorer/icons.py

```diff
@@ -144,6 +144,27 @@ def file_stack() -> svg.Svg:
     )
 
 
+def hamburger() -> svg.Svg:
+    return svg.Svg(
+        svg.Path(
+            stroke_linecap="round",
+            stroke_linejoin="round",
+            d="M3.75 6.75h16.5M3.75 12h16.5m-16.5 5.25h16.5",
+        ),
+        xmlns="http://www.w3.org/2000/svg",
```

Comment:
Why do we need this xmlns declaration for the `hamburger` and the `spinner`, but not for any other icon?